tests/messages_tests/test_cookie.py

```diff
@@ -116,15 +120,29 @@ def test_max_cookie_length(self):
         # size which will fit 4 messages into the cookie, but not 5.
         # See also FallbackTest.test_session_fallback
         msg_size = int((CookieStorage.max_cookie_size - 54) / 4.5 - 37)
+        first_msg = None
+        random.seed(42)
```

Comment:
It would be good to add some comment, it's not obvious (at least for me) why we do this.